media/css/hubs/_common.scss

```diff
@@ -13,23 +13,23 @@ $color-link-blue-dark: #175a77;
 // Section titles
 
 .section-title {
-    @include font-size(24px);
+    @include font-size-level4;
     margin: 0 0 20px;
 
     span {
         @include box-decoration-break(clone);
         background-color: #000;
         color: #fff;
         line-height: 1.3;
-        padding: 0 5px 0 60px;
+        padding: 0 5px 0 40px;
     }
 }
 
 /* -------------------------------------------------------------------------- */
 // CTA link
 
 a.cta-link {
-    @include font-size(18px);
+    @include font-size(14px);
```

Comment:
I propose adding a `font-size-level6` that is 14px and bumps up to 16px on tablet and larger (see gist https://gist.github.com/craigcook/23addac6555f239900e1e9c600553fa4). If we do that we can use it here.